testing/side-by-side.tsx

```diff
@@ -4,34 +4,32 @@ import {HeadingMedium} from "@khanacademy/wonder-blocks-typography";
 import * as React from "react";
 import ReactJson from "react-json-view";
 
-import {interactiveSizes} from "../packages/perseus/src/styles/constants";
-
 type Props = {
-    leftTitle: React.ReactNode;
-    left: React.ReactNode;
-    rightTitle?: string;
+    rendererTitle: React.ReactNode;
+    renderer: React.ReactNode;
+    JSONTitle?: string;
     jsonObject: any;
 };
 
-const SideBySide = ({
```

Comment:
After some discussion, it was determined that we really don't like this side-by-side view. I've renamed this to "SplitView" to be a little more generic, and renamed the variables accordingly. 

This moves the PerseusJSON below the Widget, which gives both the Widget and the JSON more room to breathe.